FAWE adapters don't load on Spigot 1.20

### Server Implementation

Spigot

### Server Version

1.20

### Describe the bug

Setting blocks like `//set 20` fires:
```
[16:36:20] [Server thread/INFO]: PlotSquared issued server command: //set 20
[16:36:20] [AsyncNotifyQueue - 0/INFO]: Using com.fastasyncworldedit.bukkit.adapter.NMSRelighterFactory as relighter factory.
[16:36:20] [AsyncNotifyQueue - 0/ERROR]: An unexpected error while handling a FastAsyncWorldEdit command
java.lang.NoClassDefFoundError: Could not initialize class com.sk89q.worldedit.bukkit.adapter.impl.fawe.v1_20_R1.PaperweightPlatformAdapter
	at com.sk89q.worldedit.bukkit.adapter.impl.fawe.v1_20_R1.PaperweightGetBlocks.ensureLoaded(PaperweightGetBlocks.java:362) ~[?:?]
	at com.sk89q.worldedit.bukkit.adapter.impl.fawe.v1_20_R1.PaperweightGetBlocks.getChunk(PaperweightGetBlocks.java:1005) ~[?:?]
	at com.sk89q.worldedit.bukkit.adapter.impl.fawe.v1_20_R1.PaperweightGetBlocks.getSections(PaperweightGetBlocks.java:987) ~[?:?]
	at com.sk89q.worldedit.bukkit.adapter.impl.fawe.v1_20_R1.PaperweightGetBlocks.hasSection(PaperweightGetBlocks.java:1091) ~[?:?]
	at com.fastasyncworldedit.core.extent.filter.block.CharFilterBlock.initLayer(CharFilterBlock.java:73) ~[?:?]
	at com.sk89q.worldedit.regions.Region.filter(Region.java:348) ~[?:?]
	at com.sk89q.worldedit.regions.CuboidRegion.filter(CuboidRegion.java:777) ~[?:?]
	at com.fastasyncworldedit.core.extent.filter.block.ChunkFilterBlock.filter(ChunkFilterBlock.java:88) ~[?:?]
	at com.fastasyncworldedit.core.queue.implementation.chunk.ChunkHolder.filterBlocks(ChunkHolder.java:938) ~[?:?]
	at com.fastasyncworldedit.core.queue.IQueueExtent.apply(IQueueExtent.java:148) ~[?:?]
	at com.fastasyncworldedit.core.queue.implementation.ParallelQueueExtent.apply(ParallelQueueExtent.java:108) ~[?:?]
	at com.fastasyncworldedit.core.queue.implementation.ParallelQueueExtent.setBlocks(ParallelQueueExtent.java:189) ~[?:?]
	at com.fastasyncworldedit.core.extent.PassthroughExtent.setBlocks(PassthroughExtent.java:163) ~[?:?]
	at com.sk89q.worldedit.EditSession.setBlocks(EditSession.java:1118) ~[?:?]
	at com.sk89q.worldedit.command.RegionCommands.set(RegionCommands.java:119) ~[?:?]
	at com.sk89q.worldedit.command.RegionCommandsRegistration.cmd$_set(RegionCommandsRegistration.java:602) ~[?:?]
	at org.enginehub.piston.CommandManager.execute(CommandManager.java:157) ~[?:?]
	at com.sk89q.worldedit.extension.platform.PlatformCommandManager.lambda$handleCommandOnCurrentThread$16(PlatformCommandManager.java:725) ~[?:?]
	at com.sk89q.worldedit.extension.platform.PlatformCommandManager.handleCommandTask(PlatformCommandManager.java:746) ~[?:?]
	at com.sk89q.worldedit.extension.platform.PlatformCommandManager.handleCommandOnCurrentThread(PlatformCommandManager.java:727) ~[?:?]
	at com.sk89q.worldedit.extension.platform.PlatformCommandManager.lambda$handleCommand$14(PlatformCommandManager.java:698) ~[?:?]
	at com.sk89q.worldedit.extension.platform.AbstractPlayerActor.lambda$runAction$1(AbstractPlayerActor.java:672) ~[?:?]
	at com.fastasyncworldedit.core.util.task.AsyncNotifyQueue.lambda$run$0(AsyncNotifyQueue.java:39) ~[?:?]
	at com.fastasyncworldedit.core.util.task.AsyncNotifyQueue.lambda$call$1(AsyncNotifyQueue.java:56) ~[?:?]
	at java.util.concurrent.ForkJoinTask$AdaptedCallable.exec(ForkJoinTask.java:1456) ~[?:?]
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:387) ~[?:?]
	at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1312) ~[?:?]
	at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1843) ~[?:?]
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1808) ~[?:?]
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:188) ~[?:?]
Caused by: java.lang.ExceptionInInitializerError: Exception java.lang.NullPointerException [in thread "Server thread"]
	at com.sk89q.worldedit.bukkit.adapter.impl.fawe.v1_20_R1.PaperweightPlatformAdapter.<clinit>(PaperweightPlatformAdapter.java:214) ~[?:?]
	at java.lang.Class.forName0(Native Method) ~[?:?]
	at java.lang.Class.forName(Class.java:391) ~[?:?]
	at java.lang.Class.forName(Class.java:382) ~[?:?]
	at com.sk89q.worldedit.bukkit.adapter.BukkitImplLoader.loadAdapter(BukkitImplLoader.java:170) ~[?:?]
	at com.sk89q.worldedit.bukkit.WorldEditPlugin.loadAdapter(WorldEditPlugin.java:369) ~[?:?]
	at com.sk89q.worldedit.bukkit.WorldEditPlugin.setupPreWorldData(WorldEditPlugin.java:240) ~[?:?]
	at com.sk89q.worldedit.bukkit.WorldEditPlugin.onEnable(WorldEditPlugin.java:211) ~[?:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:266) ~[spigot-api-1.20-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:342) ~[spigot-api-1.20-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:480) ~[spigot-api-1.20-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.v1_20_R1.CraftServer.enablePlugin(CraftServer.java:541) ~[spigot-1.20-R0.1-SNAPSHOT.jar:3782-Spigot-58b7c46-245f2be]
	at org.bukkit.craftbukkit.v1_20_R1.CraftServer.enablePlugins(CraftServer.java:455) ~[spigot-1.20-R0.1-SNAPSHOT.jar:3782-Spigot-58b7c46-245f2be]
	at net.minecraft.server.dedicated.DedicatedServer.e(DedicatedServer.java:220) ~[spigot-1.20-R0.1-SNAPSHOT.jar:3782-Spigot-58b7c46-245f2be]
	at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:973) ~[spigot-1.20-R0.1-SNAPSHOT.jar:3782-Spigot-58b7c46-245f2be]
	at net.minecraft.server.MinecraftServer.lambda$0(MinecraftServer.java:304) ~[spigot-1.20-R0.1-SNAPSHOT.jar:3782-Spigot-58b7c46-245f2be]
	at java.lang.Thread.run(Thread.java:1623) ~[?:?]
```

### To Reproduce

1. Build Spigot, newest version is "3782-Spigot-58b7c46-245f2be (MC: 1.20) (Implementing API version 1.20-R0.1-SNAPSHOT)" while I'm writing this

### Expected behaviour

The adapters should load and blocks should be set.

### Screenshots / Videos

_No response_

### Error log (if applicable)

_No response_

### Fawe Debugpaste

https://athion.net/ISPaster/paste/view/52f00bcabc1e4da79e57a7cdd13f2ff0

### Fawe Version

2.6.3-SNAPSHOT-471;689c7f6

### Checklist

- [X] I have included a Fawe debugpaste.
- [X] I am using the newest build from https://ci.athion.net/job/FastAsyncWorldEdit/ and the issue still persists.

### Anything else?

Currently, this can be mitigated by using Paper, and given how few people use Spigot, this is not a big concern. But restoring Spigot compat. for 1.20 would be beneficial.

--- a/worldedit-bukkit/adapters/adapter-1_19/src/main/java/com/sk89q/worldedit/bukkit/adapter/impl/fawe/v1_19_R1/PaperweightPlatformAdapter.java
+++ b/worldedit-bukkit/adapters/adapter-1_19/src/main/java/com/sk89q/worldedit/bukkit/adapter/impl/fawe/v1_19_R1/PaperweightPlatformAdapter.java
@@ -202,8 +202,8 @@ public final class PaperweightPlatformAdapter extends NMSAdapter {
             }
             try {
                 // Non-Paper
-                SERVER_LEVEL_ENTITY_MANAGER = ServerLevel.class.getDeclaredField("entityManager");
-                LEVEL_CHUNK_ENTITIES.setAccessible(true);
+                SERVER_LEVEL_ENTITY_MANAGER = ServerLevel.class.getDeclaredField(Refraction.pickName("entityManager", "P"));
+                SERVER_LEVEL_ENTITY_MANAGER.setAccessible(true);
             } catch (NoSuchFieldException ignored) {
             }
             POST_CHUNK_REWRITE = chunkRewrite;

--- a/worldedit-bukkit/adapters/adapter-1_19_3/src/main/java/com/sk89q/worldedit/bukkit/adapter/impl/fawe/v1_19_R2/PaperweightPlatformAdapter.java
+++ b/worldedit-bukkit/adapters/adapter-1_19_3/src/main/java/com/sk89q/worldedit/bukkit/adapter/impl/fawe/v1_19_R2/PaperweightPlatformAdapter.java
@@ -199,8 +199,8 @@ public final class PaperweightPlatformAdapter extends NMSAdapter {
             }
             try {
                 // Non-Paper
-                SERVER_LEVEL_ENTITY_MANAGER = ServerLevel.class.getDeclaredField("entityManager");
-                LEVEL_CHUNK_ENTITIES.setAccessible(true);
+                SERVER_LEVEL_ENTITY_MANAGER = ServerLevel.class.getDeclaredField(Refraction.pickName("entityManager", "P"));
+                SERVER_LEVEL_ENTITY_MANAGER.setAccessible(true);
             } catch (NoSuchFieldException ignored) {
             }
             POST_CHUNK_REWRITE = chunkRewrite;

--- a/worldedit-bukkit/adapters/adapter-1_19_4/src/main/java/com/sk89q/worldedit/bukkit/adapter/impl/fawe/v1_19_R3/PaperweightPlatformAdapter.java
+++ b/worldedit-bukkit/adapters/adapter-1_19_4/src/main/java/com/sk89q/worldedit/bukkit/adapter/impl/fawe/v1_19_R3/PaperweightPlatformAdapter.java
@@ -204,8 +204,8 @@ public final class PaperweightPlatformAdapter extends NMSAdapter {
             }
             try {
                 // Non-Paper
-                SERVER_LEVEL_ENTITY_MANAGER = ServerLevel.class.getDeclaredField("entityManager");
-                LEVEL_CHUNK_ENTITIES.setAccessible(true);
+                SERVER_LEVEL_ENTITY_MANAGER = ServerLevel.class.getDeclaredField(Refraction.pickName("entityManager", "L"));
+                SERVER_LEVEL_ENTITY_MANAGER.setAccessible(true);
             } catch (NoSuchFieldException ignored) {
             }
             POST_CHUNK_REWRITE = chunkRewrite;

--- a/worldedit-bukkit/adapters/adapter-1_20/src/main/java/com/sk89q/worldedit/bukkit/adapter/impl/fawe/v1_20_R1/PaperweightPlatformAdapter.java
+++ b/worldedit-bukkit/adapters/adapter-1_20/src/main/java/com/sk89q/worldedit/bukkit/adapter/impl/fawe/v1_20_R1/PaperweightPlatformAdapter.java
@@ -211,7 +211,7 @@ public final class PaperweightPlatformAdapter extends NMSAdapter {
             try {
                 // Non-Paper
                 SERVER_LEVEL_ENTITY_MANAGER = ServerLevel.class.getDeclaredField(Refraction.pickName("entityManager", "M"));
-                LEVEL_CHUNK_ENTITIES.setAccessible(true);
+                SERVER_LEVEL_ENTITY_MANAGER.setAccessible(true);
             } catch (NoSuchFieldException ignored) {
             }
             POST_CHUNK_REWRITE = chunkRewrite;

